docs/event-bus/docs/040-details-troubleshooting.md

```diff
@@ -0,0 +1,35 @@
+
+---
+title: Troubleshooting
+type: Details
+---
+## General Troubleshooting Guidelines
```

Comment:
No need for this heading - it won't display nicely on the UI. Please provide a kind of introductory sentence instead. For example:
`Follow these general troubleshooting guidelines to eliminate the most common Event Bus errors:`